src/main.js

```diff
@@ -111,6 +111,11 @@ export async function compileServices(targetName) {
 }
 
 export async function deployServices(targetName, isForced) {
+  /**
+   * Deploying is forced by default
+   */
+  isForced = true
```

Comment:
does this make any sense in the context of the signature in line 113?

Reply:
If deploy is always forced, then we should delete `isForced` from the `cli` at all.
Instead we should update `deployServicePack` method in `adapter` to:
![image](https://user-images.githubusercontent.com/25773492/99034603-a04e4580-258e-11eb-8d77-04317a696629.png)

Reply:
I agree we should remove isforced from the cli (it's redundant with this change), but not from the adapter - I think it's good to be explicit that we are going to do a full replace of the target appLoc.

Reply:
Makes sense